src/replaceRuleSelector.js

```diff
@@ -33,7 +48,9 @@ function explodeSelector(selector, options) {
 
       let newParts
       if (postSelectors.length === 0) {
-        newParts = bodySelectors.map((s) => preWhitespace + pre + s)
```

Comment:
This is actually the issue right now. Because the `pre` is a class selector and `s` is an element selector we need to flip the concatenation.